apidoc/Titanium/UI/Shortcut.yml

```diff
@@ -59,14 +63,14 @@ events:
 examples:
   - title: Simple Shortcut Example
     example: |
-        Create a shortcut item and detect its click with the `shortcutitemclick` event.
+        Create a shortcut item and detect its click with the `click` event.
 
-        ``` javascript
+        ```
```

Comment:
we want to declare the code blocks' language as we did before (it lets the doc sites syntax color the code properly)

Reply:
fixed.